pennylane/math/fn.py

```diff
@@ -339,6 +492,39 @@ def get_interface(tensor):
     return TensorBox(tensor).interface
 
 
+def marginal_prob(prob, axis):
+    """Compute the marginal probability given joint probabilities of the full system.
+
+    Args:
+        prob (tensor_like): 1D tensor of probabilities. This tensor should of size
+            ``(2**N,)`` for some integer value ``N``.
+        axis (list[int]): the axis for which to calculate the marginal

```

Comment:
```suggestion
        axes (list[int]): the axes for which to calculate the marginal
```
not sure if numpy convention is `axis`?

Reply:
Just checked, seems to be `axis`: https://numpy.org/doc/stable/reference/generated/numpy.sum.html